R/load_data.R

```diff
@@ -0,0 +1,106 @@
+# Helper functions not for export
+
+#' Check for a version of data to load
+#'
+#' This function determines if a user is requesting the latest version
```

Comment:
This is where I would specify that this is a helper function (to have it consistent with other helper functions).